packages/vats/jsconfig.json

```diff
@@ -14,5 +14,5 @@
     "strictNullChecks": true,
     "moduleResolution": "node",
   },
-  "include": ["src/**/*.js", "globals.d.ts"],
+  "include": ["src/**/*.js", "test/**/*.js", "globals.d.ts"],
```

Comment:
Whya are test files added to the main vats?

Reply:
They're not added to the vats, they're included in the package's project so that their `tsc` settings are the same.  We're not using `tsc` for bundling, so there is no other consequence.